src/pages/Defi/hooks/useCosmosStakingBalances.tsx

```diff
@@ -80,7 +81,7 @@ export function useCosmosStakingBalances({
   }, [shapeshiftValidator])
 
   const nonLoadedValidators = useAppSelector(state =>
-    selectNonloadedValidators(state, accountSpecifier),
+    selectNonloadedValidators(state, { accountSpecifier: accountSpecifier }),
```

Comment:
```suggestion
    selectNonloadedValidators(state, { accountSpecifier }),
```

Reply:
Nice one! Changed it. Wondering if we should add https://eslint.org/docs/rules/object-shorthand to our rules, WDYT? It's something that's very easily spotted when reviewing, but when you're in the middle of a refactor like this one, it's not so obvious.

Reply:
That can be arranged, @gomesalexandre!

https://github.com/shapeshift/web/pull/1663